host-controller/src/main/java/org/jboss/as/domain/controller/operations/DomainServerLifecycleHandlers.java

```diff
@@ -88,7 +88,7 @@
 
     private static final AttributeDefinition START_MODE = SimpleAttributeDefinitionBuilder.create(ModelDescriptionConstants.START_MODE, ModelType.STRING, true)
             .setDefaultValue(new ModelNode(StartMode.NORMAL.toString()))
-            .setValidator(new EnumValidator<>(StartMode.class, true, true))
```

Comment:
shouldn't you add `setAllowExpressions(true)` as the 2nd boolean is `true`?

Reply:
Fixed